### PR TITLE
dev: fix rbac doc mistake

### PR DESCRIPTION
**Pull request by imtbkcat**
### What is changed, added or deleted? 
Document of role-based access control system has some mistakes, This pr fixed them.

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1362

### Which version does your change affect? 

dev/3.0

### CODE REVIEW

**Comment by imtbkcat** [comment]
@CaitinChen

File: dev/reference/security/role-based-access-control.md [hunk 1]
```diff
@@ -338,10 +338,10 @@ SHOW GRANTS FOR 'u1'@'localhost' USING 'r1', 'r2';
 
 ### 授权表
 
-在原有的四张系统表的基础上，角色访问控制引入了三张新的系统表：
+在原有的四张系统权限表的基础上（参考：[系统权限表](/dev/reference/security/privilege-system.md#授权表)），角色访问控制引入了两张新的系统表：
```
Comment (CaitinChen) [inline dev/reference/security/role-based-access-control.md]:
```suggestion
在原有的四张[系统权限表](reference/security/privilege-system.md#授权表)的基础上，角色访问控制引入了两张新的系统表：
```

File: dev/reference/security/role-based-access-control.md [hunk 2]
```diff
@@ -384,7 +384,7 @@ select * from mysql.default_roles;
 
 ### 其他
 
-由于角色访问控制模块和用户管理以及权限管理结合十分紧密，因此需要参考一些操作的细节：
+由于基于角色的访问控制模块和用户管理以及权限管理结合十分紧密，因此需要参考一些操作的细节：
 
-- [TiDB 权限管理](/reference/security/privilege-system.md)
-- [TiDB 用户账户管理](/reference/security/user-account-management.md)
+- [TiDB 权限管理](/dev/reference/security/privilege-system.md)
```
Comment (CaitinChen) [inline dev/reference/security/role-based-access-control.md]:
```suggestion
- [TiDB 权限管理](/reference/security/privilege-system.md)
```

File: dev/reference/security/role-based-access-control.md [hunk 3]
```diff
@@ -384,7 +384,7 @@ select * from mysql.default_roles;
...
+- [TiDB 用户账户管理](/dev/reference/security/user-account-management.md)
```
Comment (CaitinChen) [inline dev/reference/security/role-based-access-control.md]:
```suggestion
- [TiDB 用户账户管理](/reference/security/user-account-management.md)
```

File: dev/reference/security/role-based-access-control.md [hunk 4]
```diff
@@ -5,9 +5,9 @@ category: user guide
 
 # 基于角色的访问控制
 
-TiDB 的角色访问控制系统参考 MySQL 的角色访问控制系统进行实现。TiDB 兼容大部分 MySQL 角色访问控制的语法。
+TiDB 的基于角色的访问控制系统参考 MySQL 的基于角色的访问控制系统进行实现。TiDB 兼容大部分 MySQL 基于角色的访问控制系统的语法。
```
Comment (CaitinChen) [inline dev/reference/security/role-based-access-control.md]:
```suggestion
TiDB 的基于角色的访问控制 (RBAC) 系统的实现类似于 MySQL 的 RBAC 系统。TiDB 兼容大部分 MySQL RBAC 系统的语法。
```

Comment (imtbkcat) [reply dev/reference/security/role-based-access-control.md]: @CaitinChen PTAL

**Pull request merged**